README.rst

```diff
@@ -1,67 +1,120 @@
-*Cellarium ML: distributed single-cell data analysis.*
+.. image:: https://cellarium.ai/wp-content/uploads/2024/07/cellarium-logo-medium.png
+   :alt: Cellarium Logo
+   :width: 180
+   :align: center
 
----------
+**Cellarium ML: a machine learning library for distributed single-cell data analysis**
```

Comment:
What about "Cellarium ML: a machine learning framework for single-cell biology"?